tests/Service/IdealTest.php

```diff
@@ -87,77 +28,30 @@ public function testPay(): void
-                "Key": "F996EE747ECD43CDA8851C5F83XXXXXX",
-                "Status": {
-                    "Code": {
-                        "Code": 190,
-                        "Description": "Success"
-                    },
-                    "SubCode": {
-                        "Code": "S001",
-                        "Description": "Transaction successfully processed"
-                    },
-                    "DateTime": "2017-03-31T09:03:45"
-                },
-                "RequiredAction": null,
-                "Services": [
-                    {
-                        "Name": "ideal",
-                        "Action": null,
-                        "Parameters": [
-                            {
-                                "Name": "customeraccountname",
-                                "Value": "J. de Tèster"
-                            },
-                            {
-                                "Name": "CustomerIBAN",
-                                "Value": "NL44RABO0123456789"
-                            },
-                            {
-                                "Name": "CustomerBIC",
-                                "Value": "RABONL2U"
-                            }
-                        ]
-                    }
-                ],
-                "CustomParameters": null,
-                "AdditionalParameters": null,
-                "RequestErrors": null,
-                "Invoice": "testinvoice 123",
-                "ServiceCode": "ideal",
-                "IsTest": true,
-                "Currency": "EUR",
-                "AmountCredit": 1,
-                "TransactionType": "C121",
-                "MutationType": 1,
-                "RelatedTransactions": [
-                    {
-                        "RelationType": "refund",
-                        "RelatedTransactionKey": "4E8BD922192746C3918BF4077CXXXXXX"
-                    }
-                ],
-                "ConsumerMessage": null,
-                "Order": null,
-                "IssuingCountry": null,
-                "StartRecurrent": false,
-                "Recurring": false,
-                "CustomerName": "J. de Tèster",
-                "PayerHash": null,
-                "PaymentKey": "AE8B6E18A2684846AAAF06A63FXXXXXX"
-            }')
-        );
-
-        $service = (new Ideal('Refund'))->setIssuer('ABNANL2A');
-        $tr = (new Transaction())->addService($service);
+        $service = (new Ideal('Pay'))->setIssuer('ABNANL2A');
+        $trPay = (new Transaction())
+            ->addService($service)
+            ->setAmount(10.00)
+            ->setInvoice('testInvoice')
+            ->setCurrency('EUR');
+        $buckaroo = new Buckaroo();
+        $buckaroo->setApiKeys(getenv('WEBSITE_KEY'), getenv('SECRET_KEY'));
+        $buckaroo->execute($trPay);
+
+        $service = (new Ideal('Refund'))
+            ->setIssuer('ABNANL2A')
+            ->setCustomerAccountName('J. de Tèster')
```

Comment:
test the fact that you do a refund, not the specific name of the customer, send only a transaction key and assert the status